js/map.js

```diff
@@ -52,15 +52,22 @@ function randomPassableTile(){
 
 function generateMonsters(){
     monsters = [];
-    let numMonsters = level+1;
+    let numMonsters = level > 5 ? level+1 : 6;
     for(let i=0;i<numMonsters;i++){
         spawnMonster();
     }
 }
 
 function spawnMonster(){
     // let monsterType = shuffle([Goose, Ant, Mushroom, Eater, Toast])[0];
-    let monsterType = shuffle([Rock, Paper, Scissors ,Rock_Plus, Paper_Plus, Scissors_Plus, Rock_Anti, Paper_Anti, Scissors_Anti])[0];
+    let monsterType = shuffle([Rock_Plus, Paper_Plus, Scissors_Plus, Rock_Anti, Paper_Anti, Scissors_Anti])[0];
     let monster = new monsterType(randomPassableTile());
```

Comment:
Any specific reason to use `let` here?

Reply:
none, why ?

Reply:
I think its a good pratice to use const as default and let just in case of needing to reassign the variable.
Check this eslint rule:
```
If a variable is never reassigned, using the const declaration is better.

const declaration tells readers, “this variable is never reassigned,” reducing cognitive load and improving maintainability.
```

https://eslint.org/docs/rules/prefer-const

I believe that it increases your code expressiveness, therefore increasing the quality of the code

Reply:
Huuummm... that is a pretty good tip, thank you very much JS wizard.